HandsOn/Group02/mappings/rules.yml

```diff
@@ -0,0 +1,41 @@
+prefixes:
```

Comment:
Hi, good first try! Syntax is perfect. But some comments about the semantics of your mappings:

I went to the ontology and I see that you have defined 10 classes, so you have to have, at least, 10 TriplesMap (not one), where you define the rule to generate the subject of each class and its associated Data and Object Properties. For this second step, predicate object maps definition (po) you have to go Data and Object Properties and see where the domains of these properties are equal to the class of the triplesMap. For example, for district, you will have something like
````
prefixes:
...
findparking: http://findyourparking.es/lcc/ontology/Parking#
mappings:
 mappingDistrict:
    sources:
      - [Parking_Slots-updated.csv~csv]
    s: http://findyourparking.es/lcc/District/$(DISTRITO)
    po:
     - [a, findparking:District]
     - [findparking:hasDistrict, $(Distrito),xsd:string] 
   (and more)
````

The source will be the same for all the triplesmaps

Reply:
Buenas noches, 
acabamos de hacer nuestra segunda versión del mapping, aunque no estamos seguros si lo hemos hecho bien ya que hemos intentado hacer el trim con el comando "sed" que nos dijiste pero no se separa por líneas.
También hemos cambiado la ontología ya que la clase Locality la hemos llamado City, también por esa razón hemos cambiado el csv y por tanto el fichero .json con los cambios.

PD: Queríamos preguntar si en cada entrega hay que indicar que integrantes del grupo han trabajado en dicha entrega, ya que hemos visto que algunos compañeros lo están indicando.  No obstante, estamos realizando todas las entregas todos los compañeros a pesar de que sea yo (el líder) el que solo hace los commits.

Un saludo.

Reply:
Hola, os acabo de dar feedback, me da la sensación de que estás confundiendo cosas, a ver si con los pasos que os he puesto conseguis tener una primera versión acorde a la ontología y luego vamos paso a paso completando el mapping, si no mandadme un email y intentamos tener una tutoria.

Por el resto está bien, no hace falta que indiquéis que hace cada uno